upgrades/upgrade_testing_framework/core/framework_state.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, state: dict):
         self.source_cluster: Cluster = None
         self.target_cluster: Cluster = None
         self.test_config: TestConfig = None
+        self.eligible_expectations = []
```

Comment:
Not a blocker, but we should indicate what type is contained within this list.

Reply:
Wasn't sure if you could do this with Python or not, will update this with next PR

Reply:
fine to have this in the next PR, but for future reference, you can do 
```
from typing import List
self.eligible_expectations: List[str] = []
```